app/code/community/Bolt/Boltpay/Model/Payment.php

```diff
@@ -778,10 +777,8 @@ public static function transactionStatusToOrderStatus( $transactionStatus ) {
                 $new_order_status = Mage_Sales_Model_Order::STATE_PROCESSING;
                 break;
             case Bolt_Boltpay_Model_Payment::TRANSACTION_PENDING:
-                $new_order_status = Mage_Sales_Model_Order::STATE_PAYMENT_REVIEW;
-                break;
             case Bolt_Boltpay_Model_Payment::TRANSACTION_REJECTED_REVERSIBLE:
-                $new_order_status = Bolt_Boltpay_Model_Payment::ORDER_DEFERRED;
```

Comment:
@johnmccombs1 While it maybe a good idea to change the order state to `Mage_Sales_Model_Order::STATE_PAYMENT_REVIEW`, it is also a good idea to keep the status as accurate as possible, so this shouldn't be changed.  It should reflect Bolt terminology.  You should change this status;s label in the database and associate its state with `Mage_Sales_Model_Order::STATE_PAYMENT_REVIEW`.

The UI, as @daisy1754 pointed out should clearly express that Bolt rejected the payment and preferably should warn that approving is not indemnified